src/services/logger.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+const betterLogging = require('better-logging');
+const { Theme } = betterLogging;
+
+betterLogging(console, {
+    color: Theme.dark,
+    saveToFile: `./logs/${Date.now()}.log`,
```

Comment:
Is it possible to make this optional via config?
Or at least have some log rotation for cleanup?